example/scalalib/native/3-multi-module/foo/native-src/htm.c

```diff
@@ -0,0 +1,547 @@
+#include "htmc.h"

```

Comment:
For the purposes of this example, we need something small that we can include in the docs and be readable for newbies. More "hello world" or "escape string" rather than "full HTML templating engine"

Reply:
As mentioned in the requirements, the code in `foo/native-src/` should depend on the code in `bar/native-src/` to demonstrate the dependency

Reply:
I get it now. `foo/native-src` should depends on `bar/native-src/` and `foo/src/` on `bar/src/` too

Before now, I thought it was `bar/native-src/` and `bar/src/` with `foo/native-src` and `foo/src/`

Reply:
I get it now. `foo/native-src` should depends on `bar/native-src/` and `foo/src/` on `bar/src/` too

Before now, I thought it was `bar/native-src/` and `bar/src/` with `foo/native-src` and `foo/src/`